packages/ListBox/src/components/Divider/Divider.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dividerCSS } from "./Divider.styles";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export default function Divider(props) {
+  return (
+    <li css={dividerCSS} aria-hidden="true">
```

Comment:
Why `aria-hidden`? I know you aren't too concerned with AT for this PR, but I am curious about this.

Reply:
remove for this PR. later we will have to figure out how to link groups from option with the screen reader :/ complex thing.